resources/visualizations/Top-Destination-Ports-By-Bandwidth.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This file should be called ```Top-Destination-Ports-By-Bandwidth-(histogram).json``` (notice to extra dash between Bandwidth and (histogram))